api/V1/Visits.ts

```diff
@@ -416,31 +423,36 @@ class Visit {
       const event = new VisitEvent(rec, track, null, taggedAs);
       this.addEvent(event);
     }
-    this.addAudioBaitEvents(rec);
   }
 
-  addAudioBaitEvents(rec) {
+  addAudioBaitEvents(allEvents: Event[]) {
     // add all audio bait events that occur within audioBaitInterval of this visit
     // and before the end of the visit
-    let events = rec.Device.Events;
-    if (!events) {
+    // let events = rec.Device.Events;
+    if (!allEvents) {
```

Comment:
You say in the function signature that the type is `Event[]`, but then you're checking to see if `allEvents` doesn't exist.  Should the type actually be something like `Event[] | undefined`?